app/admin/ip_blocks.rb

```diff
@@ -15,12 +15,16 @@
   show do
     attributes_table do
       row :ip
-      row(:reason, class: 'pre-wrap')
+      row(:reason) do
+        div(class: 'whitespace-pre-wrap') do
+          resource.reason
+        end
+      end
```

Comment:
The old syntax was raising an exception. See https://github.com/davidrunger/david_runger/pull/6585/commits/febb4e765d6072d8ae3221598f4d3b3f36c25f53 .